documentation/open-libra/validator.md

```diff
@@ -0,0 +1,266 @@
+# Things you will need:
```

Comment:
@roamingrabbit Thanks for the initiative here. One small change, can we have the file path be ./documentation/0L/... ?